<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="VISAI00215L" failures="4" tests="9" name="API Tests - MeasurementImagebyID" time="18.724" errors="0" timestamp="2023-09-07T18:41:40 IST">
  <testcase classname="v1.Get.api.MeasurementImagebyID" name="idByImage" time="1.595">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at credentails.CommonMethods.extractValuesFromImage(CommonMethods.java:95)
at v1.Get.api.MeasurementImagebyID.idByImage(MeasurementImagebyID.java:28)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- idByImage -->
  <testcase classname="v1.Get.api.MeasurementImagebyID" name="NoImageFound" time="2.445"/>
  <testcase classname="v1.Get.api.MeasurementImagebyID" name="Unauthorized" time="1.859"/>
  <testcase classname="v2.Get.api.MeasurementImagebyID" name="VeridyTheIdByImage" time="2.596">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at credentails.CommonMethods.extractValuesFromImage(CommonMethods.java:95)
at v2.Get.api.MeasurementImagebyID.VeridyTheIdByImage(MeasurementImagebyID.java:29)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- VeridyTheIdByImage -->
  <testcase classname="v2.Get.api.MeasurementImagebyID" name="testNoImageFound" time="1.474">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at credentails.CommonMethods.imageIdAssert(CommonMethods.java:164)
at v2.Get.api.MeasurementImagebyID.testNoImageFound(MeasurementImagebyID.java:49)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testNoImageFound -->
  <testcase classname="v2.Get.api.MeasurementImagebyID" name="Unauthorized" time="1.542"/>
  <testcase classname="v3.Get.api.MeasurementImagebyID" name="idByImage" time="1.686">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at credentails.CommonMethods.extractValuesFromImage(CommonMethods.java:95)
at v3.Get.api.MeasurementImagebyID.idByImage(MeasurementImagebyID.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- idByImage -->
  <testcase classname="v3.Get.api.MeasurementImagebyID" name="testNoImageFound" time="2.199"/>
  <testcase classname="v3.Get.api.MeasurementImagebyID" name="Unauthorized" time="3.313"/>
</testsuite> <!-- API Tests - MeasurementImagebyID -->
